woof-code/rootfs-petbuilds/homebank/sha256.sum

```diff
@@ -1 +1 @@
-1eece7c11b3286a53385386fb6d4a39902bffaba027cdd5a284aee58f4aa20d8  homebank-5.0.9.tar.gz
+fb9b59df8fda320f514815ce14e47b705b158ffc744e8c9c69a23edc59f41732  homebank-5.0.9.tar.gz
```

Comment:
Why not http://mirrors.ircam.fr/pub/raspbian/raspbian/pool/main/h/homebank/homebank_5.0.9.orig.tar.gz or some other mirror that points to an identical file?

Reply:
That's a great point.
I am not very well versed in public package mirrors so i defaulted to some homebank's source repository.
I wasn't even sure whether that's the official one or not.

I will amend this PR to use another mirror with the same sha256 as the original one.

Thanks! ☺👍

Reply:
It doesn't really matter if this is an official mirror or not, as long as the file hash is the same. That's why we have those hashes: to protect against accidental change of the source code we're building (and for security).